packages/orchestration/src/orchestration-api.ts

```diff
@@ -90,13 +91,28 @@ export interface Chain<CI extends ChainInfo> {
   // TODO provide a way to get the local denom/brand/whatever for this chain
 }
 
+export interface AgoricChainMethods {
```

Comment:
Distracting to have this so prominent in `orchestration-api.ts` I think it's safe to define this after use, down below.

Actually, wouldn't it be better in `local-chain-facade.js` and added those exports to `types.js`?
https://github.com/Agoric/agoric-sdk/blob/e5f1d9b5b7bcaee731400c36b41b945fd42e7674/packages/orchestration/src/types.ts#L10